ttygif.c

```diff
@@ -121,7 +121,7 @@ take_snapshot_darwin(const char *img_path, Options o)
     static char cmd [256];
 
     if (sprintf(cmd,
-            "screencapture -l$(osascript -e 'tell app \"%s\" to id of window 1') -o -m %s &> /dev/null",
+            "screencapture -l$(osascript -e 'tell app \"%s\" to id of window 1' 2> /dev/null) -o -m %s &> /dev/null",
```

Comment:
My OS X always prints a such line into stderr

    2016-03-29 10:12:12.637 osascript[5895:191750] ApplePersistence=NO

I have no idea how to turn it off